src/ansys/templates/python/common/{{cookiecutter.__project_name_slug}}/.github/workflows/ci_cd.yml

```diff
@@ -225,7 +225,7 @@ jobs:
   release-github:
     name: "Release to GitHub"
     runs-on: ubuntu-latest
-    needs: [doc-build, release-pypi-private]
```

Comment:
Also adding `build-wheelhouse` to `tests` `needs`:
```diff
-    needs: [doc-build, release-pypi-private]
+    needs: [doc-build, build-wheelhouse, release-pypi-private]
```

Because in https://github.com/ansys-internal/simba-core/actions/runs/9600322073/attempts/1 even if jobs [Build and Smoke tests (ubuntu-latest, 3.10)](https://github.com/ansys-internal/simba-core/actions/runs/9600322073/job/26476354090#logs) and [Build and Smoke tests (windows-latest, 3.10)](https://github.com/ansys-internal/simba-core/actions/runs/9600322073/job/26476354408#logs) generated [ansys-simba-core-v0.16.0-wheelhouse-ubuntu-latest-3.10](https://github.com/ansys-internal/simba-core/actions/runs/9600322073/artifacts/1621403693) and [ansys-simba-core-v0.16.0-wheelhouse-windows-latest-3.10](https://github.com/ansys-internal/simba-core/actions/runs/9600322073/artifacts/1621405186), [Release to GitHub](https://github.com/ansys-internal/simba-core/actions/runs/9600322073/job/26476379583#logs) failed to find the wheelhouse files at it was run without dependency to the `Build and Smoke tests` jobs. This is proven by re-running [Release to GitHub](https://github.com/ansys-internal/simba-core/actions/runs/9600322073/job/26477080882#logs) which then found the wheelhouse files.